Demo/Components/ChristmasWishList/ChristmasWishListDemoView.swift

```diff
@@ -62,3 +62,47 @@ extension ChristmasWishListViewModel {
         )
     }
 }
+
+#if canImport(SwiftUI) && DEBUG
+import SwiftUI
+
+struct ChristmasWishListDemoWrapperView: UIViewRepresentable {
```

Comment:
When we actually start using previews, this struct could be made into more generic `DemoWrapperView` I think.